### DISCUSSION TITLE
After using /createWorkspace in Copilot Chat, often generated files will be completely blank (about 1 in 10 of them)

### DISCUSSION THREAD

**Discussion by vuyp** [community/community#59785] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Whenever you create a workspace in Copilot Chat there's nearly always at least one file that is for some reason entirely blank even though it shouldn't be. This often completely messes up the project since all the files need to be complete for it to work. This is a great idea but right now it isn't really fully functional.

**Comment by Akash1134** [comment]
Hi @defineprogramming ,

Thank you for sharing your feedback. If you're working on VS Code IDE, please feel free to share your feedback [here](https://github.com/microsoft/vscode-copilot-release/issues).
